app/views/medical_records/_form.html.erb

```diff
@@ -31,7 +34,19 @@
     <%= form.text_field :current_state %>
   </div>
 
+  <div>
+    <%= form.label :notes, style: "display: block" %>
+    <%= form.text_area :notes, rows: 5 %>
+  </div>
+
+  <div>
+    <% if medical_record.image_location.present? %>
+      <%= image_tag(medical_record.image_location, class: "d-block w-100") %>
+    <% end %>
```

Comment:
I think the way you included the image is pretty clever. However, you don't need to attach the image to the medical record at all.
You can do here something like this:
```suggestion
    <%= image_tag("Odontogram.jpg", class: "d-block w-100") %>
```
And remove the `image_location` attribute.

Reply:
You're right, I accidentally broke something and I didn't realize what it was, so I attributed it to the image, so I made a thousand fixes, I guess they were unnecessary because it works that simple.